tests/losses/test_dice_loss.py

```
@@ -218,5 +218,57 @@ def test_script(self):
+        # Target without ignored regions (only top row evaluated)
+        input_crop = input_tensor[:, :, :1, :]
+        target_crop = target_idx[:, :, :1, :]
+        loss_fn_crop = DiceLoss(to_onehot_y=True, softmax=True, reduction="mean")
+        expected_loss = loss_fn_crop(input_crop, target_crop)
+
+        self.assertTrue(torch.allclose(loss, expected_loss))
+
+    def test_ignore_index_one_hot_target(self):
+        input_tensor = torch.randn(2, 4, 8, 8)
+        target_idx = torch.randint(0, 4, (2, 1, 8, 8))
+        target_onehot = one_hot(target_idx, num_classes=4)
```

Ruff (0.15.20)

[error] Undefined name `one_hot`

(F821)